xbmc/music/windows/GUIWindowMusicBase.cpp

```diff
@@ -807,6 +807,12 @@ void CGUIWindowMusicBase::GetContextButtons(int itemNumber, CContextButtons &but
         {
             buttons.Add(CONTEXT_BUTTON_PLAY_PARTYMODE, 15216); // Play in Partymode
         }
+        if (item->IsAudioBook())
+        {
+          int bookmark;
+          if (m_musicdatabase.GetResumeBookmarkForAudioBook(item->GetPath(), bookmark) && bookmark > 0)
+            buttons.Add(CONTEXT_BUTTON_RESUME_ITEM, 39011);
```

Comment:
i tried. i cannot comply with the request. the IContextMenu api is too limited for the use case. in particular i need to redo the listing in the media window to select the appropriate chapter in the chapter selection list, so i'd have to hop a lot of hoops without any gain (you cannot really detach this action from the window).